doc/common/README.rst

```diff
@@ -0,0 +1,42 @@
+Anuket Project
+==============
+
+Available Specifications
+------------------------
+
+-  `Reference Model <../ref_model>`__
+-  `Reference Architectures <../ref_arch>`__
+-  `Reference Implementations <../ref_impl>`__
+-  `Reference Conformances <../ref_cert>`__
+-  `Vendor Implementations <../ven_impl>`__
+
+Introduction
+------------
+
+Anuket Project :doc:`Introduction <chapter00>`
+
+Use Cases
+---------
+
+The Anuket Project addresses a wide range of use cases from the Core to the Edge of the network. Different use cases supported by the Anuket Project specifications are described in `here <../ref_model/chapters/chapter02.md#use-cases>`__.
```

Comment:
change md to rst

Reply:
This is handleg by the relative link corrector plugin. We can fix these inter document links to proper rst linking once all conversion pr-s are merged.